### PR TITLE
initial commit of helm chart and values

### PR DESCRIPTION
**Pull request by jbarchibald**
#### Purpose
move the deployment to Kubernetes

#### Changes
Added Helm chart, values and templates

#### Additional helpful information
Not ready to be merged just yet as we are still working on other facets of the deployment, however it is up and functional internally.

### CODE REVIEW

File: helm/charts/reactor-developer-docs/templates/pod-disruption-budget.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{{- if .Values.minAvailable }}
+# Pod Disruption Budgets are used to control the minimum availability of your
+# application.  If a PDB is not specified, it is valid for Kubernetes to stop
+# all of your pods in certain circumstances, such as during maintenance or if
+# the underlying infrastructure is disrupted.
+# https://kubernetes.io/docs/tasks/run-application/configure-pdb/#specifying-a-poddisruptionbudget
```
Comment (jasonwc) [inline helm/charts/reactor-developer-docs/templates/pod-disruption-budget.yaml:6]:
```suggestion
```

Might as well remove this.

File: helm/values/prod.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ingress:
```
Comment (jasonwc) [inline helm/values/prod.yaml:1]:
```suggestion
extraLabels:
  env: prod
ingress:
```

We might as well match internal convention.

**Review by jasonwc — APPROVED** [review]
Charts look good to me!

**Comment by jasonwc** [comment]
A couple other thoughts just for tomorrow:
- Consider adding `helm/` to the .dockerignore
- Decide if we want to have a stage environment. @jbarchibald and I thought it wouldn't be that valuable
- Decide if we want to support preview environments

**Comment by remear** [comment]
We never really experienced benefit from having a stage environment for this, so we tossed it. The plan was to think through previews. They'd be nice for a number of reasons, but they require some discussion.

**Review by jasonwc — APPROVED** [review]
LGTM!

**Comment by jasonwc** [comment]
Opted to move this internal after talking with Adobe Security.

**Pull request closed without merging**